### PR TITLE
Created form section

### PR DESCRIPTION
**Pull request by NgachaIan**
In this PR I added a form to my portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,7 +38,177 @@ <h1 class="header"><span>Hey there.
...
+                    </ul>
+                </div>
+            </div>
+            <div>
+                <h1 class="header-4">I'm always interested in hearing about new projects, so if you'd like to chat
+                    please get in touch.</h1>
+            </div>
+            <div>
+                <form class="form">
+                    <div class="form-group">
```
Comment (V-Blaze) [inline index.html:179]: > Collect data by using the [Formspree service](https://formspree.io/html).

- [ ] According to the project requirement, you are meant to have a post method in your form to submit the to form spree. I'll suggest you follow [this link](https://formspree.io/html), create a form spree account and get your post request URL, then add it to your form element. :smile: :wave:

File: index.html [hunk 2]
```diff
@@ -37,7 +38,177 @@ <h1 class="header"><span>Hey there.
...
+                        <input class="input-text" type="text" id="first-name" name="first-name" placeholder="Full name"
+                            required>
```
Comment (V-Blaze) [inline index.html:181]: > Name field accepts up to 30 chars.

- [ ] According to the project requirement, this name field is meant to accept a maximum of 30 characters, I'll suggest you add the `maxlength` attribute to implement this, :+1: :100:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @NgachaIan ,  @hydaline-code 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- UI looks great✔
-  Readme looks professional 👋 ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @NgachaIan  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**